docs/evaluation.md

```diff
@@ -10,100 +10,169 @@ data including inputs, outputs, and information from intermediate steps to
 evaluate the end-to-end quality of LLM responses as well as understand the
 performance of your system's building blocks.
 
-For example, if you have a RAG flow, Genkit will extract the set of documents
-that was returned by the retriever so that you can evaluate the quality of your
-retriever while it runs in the context of the flow as shown below with the
-Genkit faithfulness and answer relevancy metrics:
+### Types of evaluations
 
-```ts
-import { genkit } from 'genkit';
-import { genkitEval, GenkitMetric } from '@genkit-ai/evaluator';
-import { vertexAI, textEmbedding004, gemini15Flash } from '@genkit-ai/vertexai';
+Genkit supports two types of evaluations:
 
-const ai = genkit({
-  plugins: [
-    vertexAI(),
-    genkitEval({
-      judge: gemini15Flash,
-      metrics: [GenkitMetric.FAITHFULNESS, GenkitMetric.ANSWER_RELEVANCY],
-      embedder: textEmbedding004, // GenkitMetric.ANSWER_RELEVANCY requires an embedder
-    }),
-  ],
-  // ...
-});
-```
+1. **Inference-based evaluation**: In this type of evaluation, a system is run on a collection of pre-determined inputs and the corresponding outputs are assessed for quality. 
+    
+    This is the most common evaluation type, suitable for most use cases. This approach tests the actual output of a system for each evaluation run.
 
-**Note:** The configuration above requires installing the `genkit`,
-`@genkit-ai/googleai`, `@genkit-ai/evaluator` and `@genkit-ai/vertexai`
-packages.
+    The quality assessment can be done manually by visually inspecting the results or automated by using an evaluation metric. 
 
-```posix-terminal
-  npm install @genkit-ai/evaluator @genkit-ai/vertexai
-```
+2. **Raw evaluation**: This type of evaluation directly assesses the quality of inputs without any inference. This approach typically is used with automated evaluation using metrics. All required fields for evaluation (`context`, `output`) must be present in the input dataset. This is useful when you have data coming from an external source (eg: collected from your production traces) and you simply want to have an objective measurement of the quality of the collected data.
```

Comment:
Do you mean _quality of "outputs"_ here?

Reply:
Hmm, no? Because the _inputs_ contain the _outputs_ ;)

I can see how this can be confusing though.